### PR TITLE
WO25216: Checkin the latest translations.

### PR DESCRIPTION
**Pull request by LiweiMa**
Fixes #

## Description

Checkin the latest translation for 44 languages.

## Specific Changes

 

---

-  [ ] Testing Added

### CODE REVIEW

**Comment by corinagum** [comment]
Please add an entry to the `CHANGELOG.md`. It would also be great if you can fill out the PR template for a description of changes.

File: packages/component/src/Localization/es-ES.json [hunk 1]
```diff
@@ -1,33 +1,52 @@
 {
```
Comment (corinagum) [inline packages/component/src/Localization/es-ES.json]: @compulim @LiweiMa, are we supposed to keep the top comment about file being locked? If so, can we have it added back in to all `.json` files?

**Comment by corinagum** [comment]
@compulim, I'm unable to see the build fail details, looks like I no longer have access. Could you take a look and let us know here where the tests failed?

**Comment by tdurnford** [comment]
@corinagum Looks like its just a timeout error. Rerunning the tests should resolve the issue.

**Comment by LiweiMa** [comment]
Will create an new one which will include the comment at the top

**Pull request closed without merging**